### ISSUE TITLE
Option to make a layout the default appears only after sharing layout

### ISSUE THREAD

**Issue by tmmurali** [Murali-group/GraphSpace#171]
@DSin52 When a user creates a layout, it is initially private. For such a layout, I do not see the option to make this layout the default. This option appears only after I share the layout with the group. 

I want to be able to make a layout default just for myself. I think users would like that. Can you expose that option?

What should the behaviour be if the user then shares the layout? Will the user have to set the layout as default for the group explicitly, i.e., press the "Make layout default" button once again?

This question brings up a third issue. How do we deal with multiple users sharing a layout of the same graph and desiring to make each of their layouts the default?

**Comment by jlaw9** [comment]
Another related issue is that only the owner of the group a graph is shared to can make a shared layout default. It seems like any member of a group should be able to set a layout as default

**Comment by tmmurali** [comment]
Is it the owner of the graph or the owner of the layout that can make it
the default?

**Comment by ategge** [comment]
I wanted to add in another thought. Say I upload a network and set a layout to be the default. Jeff comes along and sets his layout to be the new default. I return to the network and no longer see my layout as the default. I might wonder (at least for a few moments) what happened. Would this be confusing? I do not know if the answer is yes or no, but it is something to think about.

**Comment by jlaw9** [comment]
@tmmurali Sorry I was mistaken in my earlier comment. Only the owner of a graph can make a layout default. In the screenshot, jamogh owns the graph and I am logged in as jeffl
![image](https://cloud.githubusercontent.com/assets/7592559/17561008/310c9aee-5ef2-11e6-9a9a-30b319ddb2a6.png)

@ategge Maybe that would be a good part of the notification framework. If the layout has changed since you last opened the graph, you'll get a notification when you open the graph

**Comment by tmmurali** [comment]
@ategge The default-layout-stomping-on-each-other should be a separate issue. I think I have discussed it with @adbharadwaj. We need a policy for this.

**Comment by adbharadwaj** [comment]
- The option to set a layout as default is available only when the layout_owner has shared the layout.
- Only layout owner can set a layout as default. This solved the `default-layout-stomping-on-each-other` problem mentioned by Murali.
